composer.json

```diff
@@ -33,21 +33,22 @@
       "App\\": "app/"
     }
   },
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/qwp6t/acorn"
-    }
-  ],
-  "minimum-stability": "dev",
   "require": {
-    "php": ">=7.1",
-    "roots/acorn": "dev-master"
+    "php": "^7.1.3",
+    "roots/acorn": "dev-master",
+    "roots/wp-blade-check": "^1.0"
```

Comment:
I would rather move wp-blade-check functionality over to acorn or refactor it and turn it into an acorn dependency.

Reply:
Gotcha. I wasn't sure what we were planning on doing with it. When I originally suggested sage-lib for 9 / Acorn for 10, @retlehs was kind of leaning towards shipping it in Sage it's self so people could opt-out of using it.